src/tacf/tacf.hpp

```diff
@@ -367,24 +364,86 @@ class Tacf : TargetedAcf
      *
      * @return A non-zero error value or zero on success.
      */
-    virtual int installAcf(const uint8_t* acf, size_t size)
+    virtual int installAcf(const uint8_t* acf, size_t size, unsigned int type,
+                           const CeLogin::AcfUserFields& acfUserFields)
     {
         if (!acf || !size)
         {
             log("acfv2 install error");
             return tacfFail;
         }
+        int rc = tacfFail;
+        switch (type)
+        {
+            case acfTypeService:
+            {
+                rc = writeFile(acf, size, acfFilePath);
 
-        int rc = writeFile(acf, size, acfFilePath);
+                // Enable the service user account using dbus interface.
+                TacfDbus().enableUser(serviceName);
 
-        // Enable the service user account using dbus interface.
-        TacfDbus().enableUser(serviceName);
+                // Unlock service user account using dbus interface.
+                TacfDbus().unlockUser(serviceName);
+                return rc;
+            }
+            break;
+            case acfTypeResourceDump:
+            {
+                std::string acfFileName = makeUniqueAcfPath();
+                rc                      = writeFile(acf, size, acfFileName);
+                if (!rc)
+                {
+                    TacfDbus().intiateResourceDump(acfFileName);
```

Comment:
Spelling.  s/intiate/initiate/ -- change both here and when the method is defined